cypress/integration/e2e/sign-in-gate.spec.js

```diff
@@ -1,5 +1,7 @@
 /* eslint-disable no-undef */
 /* eslint-disable func-names */
+/* eslint-disable mocha/no-setup-in-describe */
+import { onlyOn as onlyRunIf } from '@cypress/skip-test';
```

Comment:
Neither Jest nor Cypress support skipping tests conditionally out of the box, so this helper package seemed the best way to go. 
https://github.com/facebook/jest/issues/7245
https://github.com/cypress-io/cypress-skip-test